civis/io/_files.py

```diff
@@ -225,15 +225,18 @@ def file_to_civis(buf, name, api_key=None, client=None, **kwargs):
 
     # we should only pass _file_to_civis a file-like object that is
     # on disk, seekable and at position 0
-    if not isinstance(buf, io.BufferedReader) or buf.tell() != 0:
+    if not isinstance(buf, (io.BufferedReader, io.TextIOWrapper)) or \
```

Comment:
Can we just check `not isinstance(buf, io.IOBase) or buf.tell() != 0`?  ([IOBase docs](https://docs.python.org/3/library/io.html#io.IOBase))

```
In [16]: isinstance(io.TextIOWrapper(io.StringIO()), io.IOBase)
Out[16]: True

In [17]: isinstance(io.BufferedReader(io.StringIO()), io.IOBase)
Out[17]: True
```

Reply:
I don't think so, because `io.StringIO` and `io.BytesIO` both inherit from `io.IOBase` and the goal is to re-write those